azure-pipelines.yml

```diff
@@ -14,8 +14,10 @@ variables:
 
 steps:
 - script: |
+    set -x
     GIT_SHORT_SHA=$(echo $(Build.SourceVersion) | cut -c 1-7)
     IMAGE_NAME_WITH_TAG=$(IMAGE_NAME):$GIT_SHORT_SHA
+    set +x
```

Comment:
why this here?

Reply:
To reset that ^.

Reply:
To disable verbose set earlier as that would conflict with following echo commands used for injecting values in the environmental variables